amp/system-oracle/README.md

```diff
@@ -0,0 +1,46 @@
+# System image build with Oracle Database support
+
+
+## Prerequisites
+
+In order to build the image with Oracle Database support,
+you will need to download and add the binaries inside the directory `oracle-client-files`.
+
+Download the bellow instant clients files from the version **12.2** of oracle.
```

Comment:
below

"Download the three "Instant Client Package" files listed below to the "oracle-client-file" on your local machine"